Makefile

```diff
@@ -38,7 +38,11 @@ INSTALLER_DIR ?= ./installer
 # Tarball with the installer resources.
 INSTALLER_TARBALL ?= $(INSTALLER_DIR)/installer.tar
 # Data to include in the tarball.
-INSTALLER_TARBALL_DATA ?= charts config.yaml scripts
+INSTALLER_TARBALL_DATA ?= $(shell find $(INSTALLER_DIR) -maxdepth 1 \
+	! -path "$(INSTALLER_DIR)" \
+	! -path "$(INSTALLER_TARBALL)" \
+	! -name embed.go \
+)
```

Comment:
The tarball should always be created to avoid stable charts testing and building the project, the `tar` command only takes milliseconds to complete.

Reply:
I don't understand your comment.

- After a clone, the tarball is not present -> the archive will be created at build time
- If a user modify an installer file -> the archive will be created at build time
- If a user checks out a branch with a different installer content -> the archive will be created at build time

Reply:
Also, we have `make run` (`go run`) which needs the tarball updated beforehand.

Reply:
The target `make test` also need the tarball updated, btw.

Reply:
`make test` does update the tarball.
`make run` will with the merge of #232